### PR TITLE
OCPBUGS-8892# Add pd-balanced to sc for gcp pd

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.12+ 

Issue: https://issues.redhat.com/browse/OCPBUGS-8892

Link to docs preview: https://79275--ocpdocs-pr.netlify.app/openshift-dedicated/latest/storage/container_storage_interface/persistent-storage-csi-gcp-pd.html#persistent-storage-csi-gcp-pd-storage-class-ref_persistent-storage-csi-gcp-pd

QE review:
- [x] QE has approved this change.

Additional information:

**PTAL**: @duanwei33 @jsafrane

### CODE REVIEW

File: modules/persistent-storage-csi-gcp-pd-storage-class-ref.adoc [hunk 1]
```diff
@@ -14,7 +14,9 @@ The GCP PD CSI driver uses the `csi.storage.k8s.io/fstype` parameter key to supp
 |===
 |Parameter  |Values  |Default  |Description
 
-|`type` | `pd-ssd` or `pd-standard` | `pd-standard` | Allows you to choose between standard PVs or solid-state-drive PVs.
+|`type` | `pd-ssd`, `pd-standard`, or `pd-balanced` | `pd-standard` | Allows you to choose between standard PVs or solid-state-drive PVs.
+
+*Note*: The driver does not validate the value, thus all the possible values are accepted.
```
Comment (mburke5678) [inline modules/persistent-storage-csi-gcp-pd-storage-class-ref.adoc:19]: Can we add "proper" notes to table cells?

**Comment by mburke5678** [comment]
@lpettyjo One suggestion. Otherwise LGTM

File: modules/persistent-storage-csi-gcp-pd-storage-class-ref.adoc (hunk 1 above)
Comment (lpettyjo) [reply modules/persistent-storage-csi-gcp-pd-storage-class-ref.adoc:19]: Just decided to leave out the "note" aspect.

**Comment by jsafrane** [comment]
lgtm

**Comment by duanwei33** [comment]
LGTM

**Pull request merged**